### ISSUE TITLE
Add RANDOM ROTATION ENSEMBLE TECHNIQUE to caret 

### ISSUE THREAD

**Issue by mlnjsh** [topepo/caret#509]
If you are making a feature request or starting a discussion, you can ignore everything below and go wild =D

If you are filing a bug, make sure these boxes are checked before submitting your issue— thank you!

- [ ] Start a new R session
- [ ] Install the latest version of caret: `update.packages(oldPkgs="caret", ask=FALSE)`
- [ ] [Write a minimal reproducible example](http://stackoverflow.com/a/5963610)
- [ ] run `sessionInfo()`

### Minimal, reproducible example:
Text and example code modified from [the R FAQ on stackoverflow](http://stackoverflow.com/a/5963610)

#### Minimal dataset:
```{R}
library(caret)
set.seed(1)
dat <- twoClassSim(100)
X <- dat[,1:5]
y <- dat[["Class"]]

```
If you have some data that would be too difficult to construct using `caret::twoClassSim` or `caret::SLC14_1`, then you can always make a subset of your original data, using e.g. `head()`, `subset()` or the indices. Then use e.g. `dput()` to give us something that can be put in R immediately, e.g. `dput(head(iris,4))`

If you must use `dput(head())`, please first remove an columns from your dataset that are not necessary to reproduce the error.

If your data frame has a factor with many levels, the `dput` output can be unwieldy because it will still list all the possible factor levels even if they aren't present in the the subset of your data. To solve this issue, you can use the `droplevels()` function. Notice below how species is a factor with only one level: `dput(droplevels(head(iris, 4)))`

#### Minimal, runnable code:
```{R}
model_class <- train(
  X, y, 
  metric='ROC',
  trControl=trainControl(
    method="cv", 
    number=5,
    classProbs=TRUE, 
    summaryFunction=twoClassSummary,
    savePredictions="final")
)
print(model_class)
```

### Session Info:
```{R}
>sessionInfo()

```

You can delete the text in each section that explains how to do it correctly.
Be sure to test your 2 chunks of code in an empty R session before submitting your issue!

**Comment by mlnjsh** [comment]
Hello Max I am reading a paper 

http://stats.lse.ac.uk/fryzlewicz/rre/rre.pdf

which talks about random rotation ensemble, it will be of great help if you add it to caret

**Comment by topepo** [comment]
See [this thread](https://github.com/arnupretorius/RRotF/issues/1). It sounds like the package would not end up on CRAN and probably doesn't have much benefit so I'm going to pass on this.

**Comment by mlnjsh** [comment]
Hi Max !!! Greetings of the day!!!

Thanks for having a look.

Its fine!!!




Regards

Milan Joshi

On Friday, October 28, 2016, Max Kuhn <notifications@github.com> wrote:
> Closed #509.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.<
https://ci4.googleusercontent.com/proxy/n2VHrxCu8IeuN8FnDDMl_aQVLowyhM-uA3hORubg7tYSzNC7awdrfzezxsKivuNuDuJrfCixslcuzg0FkoIQ09WFIh0vkPRT7BS3plPp0UCGfT-MzciKwKH9IOAsnN4VSPIh5dYJuJXq0G89FCB_6ka7shZwfA=s0-d-e1-ft#https://github.com/notifications/beacon/AG4vYInlp0psq5dFbHDiKDZNKR0uWspOks5q4VftgaJpZM4KdN_B.gif>
